Objects/funcobject.c

```diff
@@ -209,6 +212,13 @@ PyFunction_SetDefaults(PyObject *op, PyObject *defaults)
     return 0;
 }
 
+void
+PyFunction_SetVectorcall(PyFunctionObject *func, vectorcallfunc vectorcall)
+{
+    func->func_version = 0;
```

Comment:
Might consider adding an `assert(func != NULL)` here for debugging convenience.

```suggestion
    assert(func != NULL);
    func->func_version = 0;
```